src/types/upload/index.ts

```diff
@@ -68,14 +68,25 @@ export interface ProgressUpdater {
     setHasLivePhotos: React.Dispatch<React.SetStateAction<boolean>>;
 }
 
+export interface ElectronFile {
+    name: string;
+    path: string;
```

Comment:
Does the path contain full path? like `C:/Users/Rushi` or path within the folder that user pointed for upload.

Reply:
Yes, full path.